aio/content/marketing/contributors.json

```
@@ -829,5 +829,13 @@
     "website": "https://thekiba.io",
     "bio": "Andrew is a software engineer using Angular and .NET. He spends most of his spare time staying up-to-date, helping other people, and experimenting with web tech.",
     "groups": ["GDE"]
+  },
+  "alvarocamillont": {
+    "name": "Alvaro Camillo Neto",
+    "picture": "alvarocamillont.png",
+    "twitter": "alvarocamillont",
+    "website": "https://alvarocamillont.dev/",
+    "bio": "Alvaro is a software engineer, speaker and instructor in Brazil. He works in the technology industry for over 10 years dedicated to the development of business solutions in a large company in the industry.",
```

```suggestion
    "bio": "Alvaro is a software engineer, speaker, and instructor in Brazil. He has worked in the technology industry for over 10 years and is dedicated to the development of enterprise business solutions.",
```

What about this?  The first three suggestions are just grammar. The last one, "enterprise", implies "a large company in the industry".

---

Perfect!